examples/byCodeSystem/README.md

```diff
@@ -0,0 +1,114 @@
+| AstraZeneca   | BARDA; OWS                                                                        | UK              | 
+| Sputnik V     | Gamaleya Res. Inst., Acellena Contract Drug R&D                                   | Russia          |
+| Sputnik Light | Gamaleya Res. Inst., Acellena Contract Drug R&D                                   | Russia          |
+| Janssen       | Janssen Vaccines Johnson & Johnson                                                | Netherlands, US |
+| CoronaVac     | Sinovac                                                                           | China           |
+| BBIBP-CorV    | Beijing Inst. of Bio. Products; Sinopharm                                         | China           |
+| EpiVacCorona  | Fed. Budgetary Res. Inst. State Res. Ctr of Vir. & Biotech.                       | Russia          |
+| Convidicea    | CanSino Biologics                                                                 | China           |
+| Covaxin       | Bharat Biotech, ICMR; Ocugen                                                      | India           |
+| WIBP-CorV     | Wuhan Inst. of Bio. Products; Sinopharm                                           | China           |
+| CoviVac       | Chumakov Fed. Sci. Ctr for R&D of Imm. and Biological Products                    | Russia          |
+| ZF2001        | Anhui Zhifei Longcom Biopharm.; Inst. of Microbio. of the Chinese Academy of Sci. | China           |
+| QazVac        | Res. Inst. for Biological Safety Problems                                         | Kazakhstan      |
+| Unnamed       | Minhai Biotechnology Co.; Kangtai Biological Products Co. Ltd.                    | China           |
+| Novavax       | Novavax                                                                           | US              |
+
+Also added Novavax vaccine, although it is not yet approved as of Jun 9th, 2021.
+
+## Vaccine Codes
+
+Blank means values are missing, and " " indicates same value for each row.
+
+| Vaccine           | GTIN            | CVX      | AIR      | SNOMED CT | ICD-11    | ATC/DDD  |
+| :---------------: | :-------------: | :------: | :------: | :-------: | :-------: | :------: |
+| Pfizer            |                 | 208      | COMIRN   | 840534001 |  XM0GQ8   | J07BX03  |
+| Moderna           | 30380777700688  | 207      |          |   "   "   |  XM0GQ8   |  "   "   |
+| AstraZeneca       |                 | 210      | COVAST   |   "   "   |  XM9QW8   |  "   "   |
+| Sputnik V         |                 | 213      |          |   "   "   |  XM9QW8   |  "   "   |
+| Sputnik Light     |                 | 213      |          |   "   "   |  XM9QW8   |  "   "   |
+| Janssen           | 00359676580157  | 212      |          |   "   "   |  XM9QW8   |  "   "   |
+| Sinovac           |                 | 213      |          |   "   "   |  XMLNL1   |  "   "   |
```

Comment:
change XMLNL1 to XM1NL1
see https://icd.who.int/dev11/l-m/en#/http%3a%2f%2fid.who.int%2ficd%2fentity%2f1104808441

Reply:
I just fixed this in next commit, thank you for finding it